DEVELOPMENT.md

```diff
@@ -55,6 +55,13 @@ development command line options.
   If handling of metadata has changed while developing, set this env var to
   `clear` to have cache `clear()`ed before use.
 
+- `DANDI_INSTANCEHOST` -- defaults to `localhost`. Point to host/IP which hosts
```

Comment:
Mention that, when this is set, `DANDI_REUSE_LOCAL_DOCKER_TESTS_API_KEY` should be set as well.

Reply:
although in my case and probably thus "typically" it is the case -- it not generally so since we could have some instance running e.g. on public IP, so we could still `docker-compose up` it and interact with it.

Reply:
but I will add a comment that the API_KEY should typically be set as well.